package.json

```diff
@@ -33,7 +34,7 @@
   "license": "MIT",
   "dependencies": {
     "JSONStream": "^1.1.4",
-    "joi": "^10.0.2",
+    "joi": "^13.0.2",
```

Comment:
joi 13 requires node 8. that's why the travis tests are failing. can you bump the node version in the `.travis.yaml` file